src/static/containers/CardList.js

```diff
@@ -21,15 +29,24 @@ const CardList = ({ usercardsFetch }) => {
 
 CardList.propTypes = {
     usercardsFetch: PropTypes.instanceOf(PromiseState).isRequired,
+    refreshUserCards: PropTypes.func.isRequired,
 };
 
+const usercardsFetch = token => ({
+    url: '/api/v1/user_cards/',
+    force: true,
+    headers: {
+        Accept: 'application/json',
+        Authorization: `Token ${token}`,
+    },
+});
+
 export default connect(({ token }) => ({
-    usercardsFetch: {
-        url: '/api/v1/cards/user_cards/',
-        force: true,
-        headers: {
-            Accept: 'application/json',
-            Authorization: `Token ${token}`,
+    usercardsFetch: usercardsFetch(token),
+    refreshUserCards: () => ({
```

Comment:
Maybe add a comment to explain that it overrides the previous func in order to force refreshing inside the component? For people who are not familiar with react-refetch